coremltools/models/neural_network/spec_inspection_utils.py

```diff
@@ -122,50 +132,55 @@ def summarize_neural_network_spec(mlmodel_spec):
     elif mlmodel_spec.HasField('neuralNetworkRegressor'):
         nn = mlmodel_spec.neuralNetworkRegressor
 
-    layers = [summarize_network_layer_info(layer) for layer in nn.layers] if nn != None else None
+    layers = [_summarize_network_layer_info(layer) for layer in nn.layers] if nn != None else None
     return (inputs, outputs, layers)
 
-def prRed(skk, end=None):
+def _prRed(skk, end=None):
     print("\033[91m {}\033[00m".format(skk), end=end)
 
-def prLightPurple(skk, end=None):
+def _prLightPurple(skk, end=None):
```

Comment:
Just a comment, Not related to this PR. 

Recently found that `print` with `end` argument is only available in python 3.x. 

So, python 2.x users won't be able to use this awesome coding style printing

Reply:
Although it still seems to work fine in python 2

Reply:
I see, we seem to import print function from future. 

Never mind. Sorry for the noise.